RELEASE_NOTES.md

```diff
@@ -27,240 +27,185 @@ will not be possible to downgrade from 3.0 to 2.6.x any more.
-                                  best experience if Roboto font installed
+                                  best experience with Roboto font installed
 
      - darktable-elegant-dark   : lighter version
      - darktable-elegant-grey   : even lighter version
 
-     - darktable-icons-dark     : lighter version
-     - darktable-icons-grey     : even lighter version
+     - darktable-icons-dark     : lighter version. with IOP icons
+     - darktable-icons-grey     : even lighter version, with IOP icons
 
-  Note that this requires Gtk 3.22 or higher to work properly.
+  Note that the new GUI requires Gtk+ 3.22 or higher to work properly.
 
-- The color picker on the blend, tone curve, color zones and fill
-  light modules now allows to select an area by <kbd>Ctrl+click</kbd> on it.
+- The color picker on the tone curve, color zones and fill light modules,
+  as well as the parametric mask controls, now allows you to select an
+  area when enabled by using <kbd>Ctrl+click</kbd> on the button.
 
-- Add undo/redo support in lighttable for tags, color labels, rating,
+- Added undo/redo support in lighttable for tags, color labels, ratings,
   metadata, deleted history stack, pasted history stack and applied
-  style.
+  styles.
 
   IMPORTANT:
-  As <kbd>Ctrl-z</kbd> is commonly used for undo/redo this is what is used
-  on the lighttable. This conflicts with the previous <kbd>Ctrl-z</kbd> which
-  was assigned to preview with focus point. The preview (previously <kbd>z</kbd>)
-  and preview with focus detection (previously <kbd>Ctrl-z</kbd>) are now assigned
-  respectively to <kbd>w</kbd> and <kbd>Ctrl-w</kbd>.
+  The 'preview' (previously <kbd>Z</kbd>) and 'preview with focus detection'
+  (previously <kbd>Ctrl+Z</kbd>) actions are now assigned respectively to
+  <kbd>w</kbd> and <kbd>Ctrl+W</kbd> in order to follow the convention
+  of using <kbd>Ctrl+Z<kbd> for the "undo" function.
 
 - A new timeline view has been introduced in the lighttable.
 
-- A new lighttable mode 'culling' is added. It displays a fixed number
-  of consecutive images, starting from the first selected image and
-  allows to pan & zoom them.  It can be navigated with the mouse wheel
-  and keyboard and the number of displayed images can be set with an
-  entry at the bottom.
+- A new 'culling' mode has been added added to the lightable view. It
+  displays a fixed number of consecutive images starting from the first
+  selected, and allows you to pan & zoom them. The number of displayed images
+  can be set by the user, and they can be navigated with the mouse wheel
+  and keyboard.
 
-- A quite extensive rewrite of the lighttable has been made to improve
-  drastically the overall performance. This also includes the
-  filmstrip which has also received a big rewrite for the same reasons. The
-  lighttable is now usable on 4K and 5K monitors.
+- A quite extensive rewrite of the lighttable (including the film strip) has
```

Comment:
Lighttable has timeline view and  darkroom has filmstrip view.

Reply:
...and I don't generally use either one... thanks!